File: dataset_reduzido/codenet/Java/Code/s936984238.java
import java.util.*;
public class Main {
	static Scanner sc = new Scanner(System.in);
	/**
	 * @param args
	 */
	static int[] score;
	static int number;
	public static void main(String[] args) {
		// TODO ツ篠ゥツ督ョツ青カツ青ャツつウツづェツつスツδソツッツドツ・ツスツタツブ
		while(read()){
			slove();
		}
	}
	static boolean read(){
		number = sc.nextInt();
		if(number == 0)
			return false;
		
		score = new int[number];
		for(int i = 0; i < number; i++){
			score[i] = sc.nextInt();
		}
		return true;
	}
	static void slove(){
		java.util.Arrays.sort(score);
		int sum = 0;
		for(int i = 1; i < number-1; i++){
			sum = sum + score[i];
		}
		System.out.println(sum/(number-2));
	}
}

File: output/LLaMa_repaired_1/codenet/Java/Python/s936984238.py
def read():
    number = int(input())
    if number == 0:
        return False
    score = []
    for _ in range(number):
        score.append(int(input()))
    return score

def slove(score):
    score.sort()
    sum = sum(score[1:-1])
    print(sum/(len(score)-2))

while True:
    if not score := read():
        break
    slove(score)